test/unit/specs/data/json-update-builder.js

```diff
@@ -0,0 +1,189 @@
+import { describe, it } from 'mocha'
+
+import Backendless from '../../helpers/sandbox'
+import { expect } from 'chai'
+
+describe('<Data> JSON Update Builder', function() {
+  let jsonUpdateBuilder
+
+  beforeEach(() => {
+    jsonUpdateBuilder = Backendless.Data.JSONUpdateBuilder
+  })
+
+  it('should have null as default values', async () => {
+    const jsonUpdateBuilderInstance = new jsonUpdateBuilder()
+
+    expect(jsonUpdateBuilderInstance.args).to.be.equal(null)
+    expect(jsonUpdateBuilderInstance.operationName).to.be.equal(null)
+  })
+
+  it('set', async () => {
+    const jsonUpdateSet = jsonUpdateBuilder.SET()
+
+    expect(jsonUpdateSet.args).to.be.eql({})
+    expect(jsonUpdateSet.operationName).to.be.equal('JSON_SET')
+  })
+
+  it('set with args', async () => {
+    const jsonUpdateSet = jsonUpdateBuilder.SET()
+      .addArgument('$.letter', 'b')
+      .addArgument('$.number', 36)
+      .addArgument('$.state', true)
+      .addArgument('$.colours[0]', null)
+      .addArgument('$.innerObject', { a: 'b' })
+      .addArgument('$.innerArray', [4, 3, 2])
+
+    expect(jsonUpdateSet.args).to.eql({
+      '$.letter'     : 'b',
+      '$.number'     : 36,
+      '$.state'      : true,
+      '$.colours[0]' : null,
+      '$.innerObject': { a: 'b' },
+      '$.innerArray' : [4, 3, 2]
+    })
+    expect(jsonUpdateSet.operationName).to.be.equal('JSON_SET')
+  })
+
+  it('set with args and create', async () => {
+    const jsonUpdateSet = jsonUpdateBuilder.SET()
+      .addArgument('$.letter', 'b')
+      .addArgument('$.number', 36)
+      .addArgument('$.state', true)
+      .addArgument('$.colours[0]', null)
+      .addArgument('$.innerObject', { a: 'b' })
+      .addArgument('$.innerArray', [4, 3, 2])
+      .create()
```

Comment:
remove the `.create()` and add only one `it` to check that `create` return the save as `toJSON` method

Reply:
toJson returns a string but create - an object

Reply:
why `toJSON` should return a string?